cmd/run.go

```diff
@@ -102,6 +110,10 @@ func start(cliCtx *cli.Context) error {
 		}
 	}
 
+	if c.Metrics.Enabled {
+		go startMetricsHttpServer(c)
```

Comment:
I see that this will expose an endpoint where metric can be checked. Am I right? I thought metric would be sent to a central server where all the metrics would be collected. Why did you choose this polling strategy??

Reply:
@ARR552 there's been quite a discussion about whether to push or pull metrics at the beginning of the implementation. The main concern was about the jsonrpc service that can scale horizontally.
Finally we decided together with @xavier-romero to keep the pulling strategy and have the scaling problem resolved on the metrics server side (through service discovery or other custom mechanism).

Prometheus scraper collects metrics by pulling that endpoint on services it is aware of. The other way around would have required to set up a metrics push-gateway and [it seems to not being recommended as a generic solution](https://prometheus.io/docs/instrumenting/pushing/).